src/MixedModels.jl

```diff
@@ -82,6 +83,7 @@ export @formula,
        refit!,
        residuals,
        response,
+       shortestCovInt,
```

Comment:
I'm quite late to the party, but this name doesn't follow the Julia conventions (lowercase).